tests/dynamics/test_circular_singles_encounters_prograde.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env python3
+"""Test for circular_singles_encounters_prograde function
+    in physics/dynamics.py
+"""
+######## Imports ########
+#### Standard Library ####
+import time
+#### Third-Party ####
+import numpy as np
+#### Homemade ####
+#### Local ####
+from mcfacts.physics.dynamics import circular_singles_encounters_prograde
+
+######## Setup ########
+INPUT_DATA = {
```

Comment:
Please utilize the `INPUT_PARAMETERS` dictionary in `McFACTS/conftest.py` for these values.